mylyn.builds/org.eclipse.mylyn.jenkins.tests/META-INF/MANIFEST.MF

```diff
@@ -16,7 +16,11 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="0.0.0",
  org.eclipse.search;bundle-version="0.0.0",
  org.eclipse.ui;bundle-version="0.0.0",
  org.eclipse.ui.workbench.texteditor;bundle-version="0.0.0",
- org.junit;bundle-version="4.8.2"
+ junit-jupiter-api;bundle-version="5.10.3",
```

Comment:
I would recommend to prefer "Import-Package" for 3rd party libs

Reply:
Sure. 
I wasn't sure about that so I just followed what was done with JUnit 4

Reply:
So I tried "importing" and had to add the libraries to the .target file with older versions. So I don't know what is the best solution?

I'm still trying to figure out how all this works. 
What are the pros and cons of using the versions made available implicitly vs the ones from the orbit/drop/??? sites